models/_sector/referral/rewards/platforms/zora_zora_rewards.sql

```diff
@@ -0,0 +1,16 @@
+{{ config(
+    schema = 'zora_zora',
```

Comment:
i assume it's to match decoded naming standard, but still a bit odd to see `zora_zora` instead of just `zora`

is it worth simplifying that naming standard universally, or better to indicate it's zora protocol on zora blockchain to differentiate?

Reply:
I don't like it either.. But still think it's the best we can do for the reason you mentioned above.